src/utils/helper.ts

```diff
@@ -2,3 +2,6 @@ export const isEurope = () => {
   const offset = new Date().getTimezoneOffset();
   return offset <= 0 && offset >= -180;
 };
+
+export const hypehnate = (str: string) =>
```

Comment:
A small spelling mistake and I would set a default value for `str` to avoid a runtime error if `str` is `undefined`.

```suggestion
export const hyphenate = (str: string = "") =>
```